statchatexpress/package.json

```diff
@@ -14,10 +14,12 @@
     "cors": "^2.8.5",
     "dotenv": "^16.3.1",
     "express": "^4.18.2",
+    "express-session": "^1.17.3",
     "morgan": "^1.10.0",
     "pg": "^8.11.1",
     "pg-hstore": "^2.3.4",
     "sequelize": "^6.32.1",
-    "sequelize-cli": "^6.6.1"
+    "sequelize-cli": "^6.6.1",
+    "uuid": "^9.0.0"
```

Comment:
what's this used for?

Reply:
uuid  was to handle user sessions but i didnt get to that yet and i might not be using that.